ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -171,7 +171,7 @@ statsFunctionName
     ;
 
 percentileAggFunction
-    : PERCENTILE '<' value=integerLiteral '>' LT_PRTHS aggField=fieldExpression RT_PRTHS
+    : PERCENTILE LESS value=integerLiteral GREATER LT_PRTHS aggField=fieldExpression RT_PRTHS
```

Comment:
this seems unrelated, but I assume it's tested and okay to push?

Reply:
Yes. In addition, the percentile function is not actually supported -- it's in the parser, but fails semantic analysis.